docs/specs/component.md

```diff
@@ -84,6 +84,22 @@ representing multiple endpoints. If a component uses multiple options to
 automatically build the endpoint, then the `endpoint(s)` option MUST
 override that process.
 
+#### `listen`
+
+When a component listens for incoming connections, it SHOULD expose a `listen` configuration option that takes
+a `string` representing an address with `<protocol>:<address>`.
```

Comment:
This is the user facing type, right? Can we clarify that we should have an internal type to represent these? (I'm assuming we'd want a type for this)

Reply:
🤔 I think the internal implementation is outside of the scope of this spec which just focuses on the UX for users.

Reply:
Fair enough.